src/risedevtool/src/bin/risedev-dev.rs

```diff
@@ -309,9 +306,14 @@ fn task_main(
                 let mut ctx =
                     ExecuteContext::new(&mut logger, manager.new_progress(), status_dir.clone());
                 MySqlService::new(c.clone()).execute(&mut ctx)?;
-                let mut task =
-                    risedev::ConfigureTcpNodeTask::new(c.address.clone(), c.port, c.user_managed)?;
-                task.execute(&mut ctx)?;
+                if c.user_managed {
```

Comment:
TBH, the most elegant way should be connecting to the database programmingly. However, this will introduce extra dependencies to RiseDev so I slightly prefer the log way.

Reply:
😃  The log way LGTM, which is simple and robust enough I think.